xicam/gui/bluesky/search.py

```diff
@@ -224,7 +251,7 @@ def show_results(self):
         self.show_results_event.clear()
         t0 = time.monotonic()
         counter = 0
-
+        msg.showBusy()
```

Comment:
I think show_results should always be really quick < 5ms. check_for_new_entries is the method that takes a while, maybe this should be in the check_for_new_entries method?